src/components/NewsGrid/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {SimpleGrid} from '@chakra-ui/react';
+import NewsCard from "../NewsCard";
+
+const NewsGrid = () => (
+    <SimpleGrid columns={{base: 1, sm: 2, md: 4}} spacing={3}>
```

Comment:
[MINOR] Was wondering if it could be better to space it out to 
base:1 sm:2 md:3

Reply:
How about `base: 1, sm: 2, md: 3, lg: 4` ?

I think when the window is maximized, there's too much white-space if we only have 3 columns

**three:**
![three](https://user-images.githubusercontent.com/51832617/121766386-44d53d80-cb06-11eb-8e8e-d0723da77570.png)


**four:**
![four](https://user-images.githubusercontent.com/51832617/121766387-47d02e00-cb06-11eb-86b5-bdf0649bce14.png)

Reply:
This looks good :))